index.js

```diff
@@ -129,7 +189,18 @@ function filterByGenre() {}
       }
     ];
  */
-function getAllMoviesReleasedAtOrBeforeYear() {}
+function getAllMoviesReleasedAtOrBeforeYear(movies, year) {
+let released = [];
+
+for(let i = 0; i < movies.length; i++){
+  
+  dateArr = movies[i].released.split(" ");
```

Comment:
I might change this to `currentReleaseYear` and set it equal to `movies[i].released.split(" ")[2]`